theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer-toolbar-commands.ts

```diff
@@ -14,6 +14,18 @@ export namespace TraceViewerToolbarCommands {
         iconClass: 'fa fa-minus-square-o fa-lg',
     };
 
+    export const UNDO: Command = {
+        id: 'trace.viewer.toolbar.undo',
+        label: 'Undo',
+        iconClass: 'fa fa-undo fa-lg',
+    };
+
+    export const REDO: Command = {
+        id: 'trace.viewer.toolbar.redo',
+        label: 'Redo',
+        iconClass: 'fa fa-repeat fa-lg',
+    };
+
```

Comment:
Let me know if the icons look OK to you.

Reply:
I like those icons, but the spacing looks weird with them.  The FontAwesome icons for some reason float to the left, while the VSCode icons are properly centered.  They look off when next to each other, in my opinion.

![image](https://user-images.githubusercontent.com/98342456/165152581-a217386c-2b84-40b7-8c0f-89600980e6dc.png)